### DISCUSSION TITLE
npx wagmi generate "Invalid literal value"

### DISCUSSION THREAD

**Discussion by rhlsthrm** [wevm/wagmi#2993] [Q&A] [answered]
I'm using a deployed and verified contract on Goerli, and getting this error:

```
npx wagmi generate          

✔ Validating plugins
⠇ Resolving contracts
Invalid response
- Invalid literal value, expected "OK" at `message`
```

Any idea how I can find out what's going on?

**Comment by tmm** [comment]
Can you share your `wagmi.config.ts`? My guess is you don't have a API key set.

**Reply by rhlsthrm** [reply]
I have an API key. It works when I use a different contract address but just not this one for some reason.

**Reply by tmm** [reply]
If you want help, share your code. Otherwise, I'm just guessing 😃

**Reply by rhlsthrm** [reply]
Here's the code, it's quite simple:

```ts
import { defineConfig } from "@wagmi/cli";
import { etherscan, react } from "@wagmi/cli/plugins";
import { goerli } from "wagmi/chains";

export default defineConfig({
  out: "src/generated.ts",
  plugins: [
    etherscan({
      apiKey: process.env.ETHERSCAN_API_KEY!,
      chainId: goerli.id,
      contracts: [
        {
          name: "VapeGame",
          address: {
            [goerli.id]: "0x64d80a46C4183A3B9CBca6dAEA8B3397C43FA13A",
          },
        },
      ],
    }),
    react(),
  ],
});
```

**Reply by rhlsthrm** [reply]
OK weird, if I hardcode the ETHERSCAN_API_KEY it works. It just doesn't seem to pick it up from .env.

**Reply by tmm** [reply, marked as answer]
The CLI doesn't automatically load from `.env`. You can use [`loadEnv`](https://wagmi.sh/cli/configuration/configuring-cli#environment-variables) for that.
